### PR TITLE
refactor store for multiply accounts management

### PR DESCRIPTION
**Pull request by ttaoS**

### CODE REVIEW

File: src/ts/components/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -100,6 +117,9 @@ class Dashboard extends React.Component<IDashboardProps, IDashboardState> {
           <StyledButton onClick={this.handleClick}>
             {t('logout')}
           </StyledButton>
+          <StyledButton onClick={this.handleCreateAccountClick}>
+            create new account
```
Comment (ttaoS) [inline src/ts/components/dashboard/Dashboard.tsx]: for testing only, we will use i18n for text later.

File: src/ts/background/store/wallet.ts [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 
-export interface IAccount {
+export interface IWallet {
   locked: boolean,
   created: boolean,
   newPassword?: string,
   newPhrase?: string,
   newAccountName?: string
+  accounts?: IAccount[]
   currentAddress?: string
```
Comment (fyang1024) [inline src/ts/background/store/wallet.ts]: as discussed, currentAccount replacing currentAddress and currentName is probably cleaner

File: src/ts/components/account/ConfirmPhrase.tsx [hunk 3]
```diff
@@ -56,21 +56,32 @@ class ConfirmPhrase extends React.Component<IConfirmPhraseProps, IConfirmPhraseS
...
         // update redux store state
         this.props.setLocked(false)
         console.assert(kp.length === 0, 'Should be an empty array')
-        if (accountStatus.newPhrase) {
-          createAccount(accountStatus.newPhrase, accountStatus.newAccountName).then(keyringPair => {
+        if (wallet.newPhrase) {
+          createAccount(wallet.newPhrase, wallet.newAccountName).then(keyringPair => {
             console.log('Account created! ', keyringPair)
             this.props.setCreated(true)
             this.setState({ keyringPair: keyringPair })
```
Comment (fyang1024) [inline src/ts/components/account/ConfirmPhrase.tsx]: `this.setState({ keyringPair })` shortcut might be used

File: src/ts/components/dashboard/Dashboard.tsx [hunk 4]
```diff
@@ -1,19 +1,21 @@
 import * as React from 'react'
 import { getAccounts, lockWallet } from '../../services/keyring-vault-proxy'
-import { LOGIN_ROUTE } from '../../constants/routes'
+import { GENERATE_PHRASE_ROUTE, LOGIN_ROUTE } from '../../constants/routes'
 import { RouteComponentProps, withRouter } from 'react-router'
 import { Button as StyledButton, ContentContainer, Section } from '../basic-components'
 import { IAppState } from '../../background/store/all'
 import { connect } from 'react-redux'
-import { setCurrentAddressAndName } from '../../background/store/account'
```
Comment (fyang1024) [inline src/ts/components/dashboard/Dashboard.tsx]: `setCurrentAccount` may be easier to read

Comment (ttaoS) [reply src/ts/components/dashboard/Dashboard.tsx]: ? it is now `setCurrentAccount `

Comment (fyang1024) [reply src/ts/components/dashboard/Dashboard.tsx]: sorry, page not refreshed. All good

File: src/ts/components/account/ConfirmPhrase.tsx (hunk 3 above)
Comment (ttaoS) [reply src/ts/components/account/ConfirmPhrase.tsx]: yes. updated.

**Pull request merged**